### ISSUE TITLE
French translations updates

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#195]
```
I have looked at the French translations, and updated some files.
Some of my changes are simple grammatical or typos fixes, some more 
typographical (despite what many French people say, upper case letters are 
accentuated). I also used the upper and lower cases oe ligature (UTF-8 does 
handles it perfectly) but I don't know if all devices can display them.
I have also slightly changed a few items names for emphasis (renamed ring of 
lesser shadow from « Anneau des petites ombres »  to « Anneau de la Moindre 
Ombre » for example), so I guess the original translator may not agree.

I will have a look at some non-translated files if possible. Is there already 
some work done on them ?

I would like to attach a patch file with my updates to this issue, but the line 
at the bottom of the form I am editing reads: "Issue attachment storage quota 
exceeded". How could I send my fixes ?
```

Original issue reported on code.google.com by `oxy...@gmail.com` on 23 May 2011 at 7:57

**Comment by GoogleCodeExporter** [comment]
```
Committed in r98. Thanks!

(About the issue attachment storage, I sent a question to Google about 
increasing our storage quota)
```

Original comment by `oskar.wi...@gmail.com` on 25 May 2011 at 7:56
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:52
- Changed state: **Done**
